frontend/javascripts/pages/frontpage/pricing_view.js

```diff
@@ -0,0 +1,360 @@
+      </div>
+      <h4 style={{ marginTop: 20 }}>{title}</h4>
+      <p style={{ marginTop: 20 }}>{subTitle}</p>
+    </div>
+  </Col>
+);
+
+const PricingView = () => (
+  <>
+    <div className="container" style={{ paddingTop: 20 }}>
+      <Row gutter={16} type="flex" style={{ marginTop: 60, marginBottom: 60, textAlign: "center" }}>
+        <Col
+          lg={{ span: 16, offset: 4 }}
+          md={{ span: 16, offset: 4 }}
+          sm={{ span: 22, offset: 2 }}
+          xs={{ span: 22, offset: 2 }}
+        >
+          <h1>Hosting Solutions</h1>
+          <div>
+            <h4>
+              Our webKnossos hosting solutions get you started with webKnossos today. <br />
+              Fast, functional, and flexible. Trusted by leading research labs.
+            </h4>
+          </div>
+        </Col>
+      </Row>
+      <Row gutter={16} type="flex">
+        <FeatureItem
+          title="High Speed Skeleton Annotations"
+          subTitle="Use our published Flight mode to quickly capture high-quality skeletons"
+          iconUrl="/images/high-speed-tracing-icon.svg"
+        />
+        <FeatureItem
+          title="Volume Annotation"
+          subTitle="Create and proof-read dense volume annotations"
+          iconUrl="/images/volume-annotation-icon.svg"
+        />
+        <FeatureItem
+          title="Simple Collaboration"
+          subTitle="webKnossos runs in the browser. Share links to annotations and publish datasets"
+          iconUrl="/images/collaboration-icon.svg"
+        />
+      </Row>
+
+      <Row gutter={16} type="flex" style={{ marginTop: 90, marginBottom: 100 }}>
+        <PricingColumn title="Public Hosting" iconUrl="/images/public-hosting-icon.svg" price={0}>
+          {[
+            "Try out webKnossos for free with any published datasets on webknossos.org",
+            "Unlimited accounts for your organization",
+            "Suggest new datasets for publication",
+            "Community Support",
+            "All other wK features",
+            <Link to="/onboarding" key="link-to-onboarding">
+              <Button size="large" style={{ marginTop: 20 }}>
+                Create a Free Account
+              </Button>
+            </Link>,
+          ]}
+        </PricingColumn>
+
+        <PricingColumn
+          title="Premium Hosting"
+          iconUrl="/images/premium-hosting-icon.svg"
+          price={250}
+          contentStyle={{
+            marginTop: "-8%",
+            height: "110%",
+            backgroundColor: bgColorDark,
+            color: "white",
+          }}
+        >
+          {[
+            "Upload and work with your data hosted on scalable minds servers",
+            "From 100GB of data storage**",
+            "Managed monitoring and backups",
+            "Help with dataset upload and conversion ",
+            "Email support",
+            "Everything from the Public Hosting",
+            <a href="mailto:hello@scalableminds.com" key="email-button">
```

Comment:
Can we add some analytics to these buttons, so that we know whether people clicked and didn't send emails.

Reply:
@daniel-wer @philippotto Didn't you guys add custom analytics to wK lately? Can we use that here?

Reply:
Yes, use `import { trackAction } from "oxalis/model/helpers/analytics";` to import and then `trackAction("Click get in touch");` or whatever string you want. This will then show up in analytics as an event.